packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -86,11 +85,13 @@ html.interface-interface-skeleton__html-container {
 	left: 0;
 	background: $white;
 	color: $gray-900;
+	width: 0;
```

Comment:
I think this is really close. Something about this style is breaking the inspector panel on both the site editor and the post editor. You can see that here:

<img width="405" alt="image" src="https://user-images.githubusercontent.com/1123119/95783310-d1d1a980-0c8e-11eb-9c17-0a21c71cdc29.png">

Reply:
Awesome catch, I'll need to address that! Thank you.

Reply:
Alright this should be fixed.